rules/power-point-comments/rule.md

```diff
@@ -0,0 +1,46 @@
+---
+type: rule
+title: Do you use PowerPoint comments for feedback?
+uri: power-point-comments
+authors:
+  - title: Isabel Sandstroem
+    url: https://www.ssw.com.au/people/isabel-sandstroem
+  - title: Daniel Mackay
+    url: https://ssw.com.au/people/daniel-mackay/
+created: 2024-02-22T10:00:00.000Z
+guid: c7a5a8e3-7e5d-4e3f-875f-8e0f2c98b3a9
+---
+
+Imagine you've just finished a draft of your PowerPoint presentation and you send it off to your colleagues for feedback. You receive it back with a mix of emails, separate documents, and verbal comments - a nightmare to consolidate and action. 
+This is where PowerPoint comments come into play, providing an effective workflow for gathering and acting on feedback.
+
+<!--endintro-->
+
+### The Power of PowerPoint Comments
+
+PowerPoint comments allow you to add notes directly on the slides, offering a simple and integrated way to give and receive feedback. This method keeps all the feedback in one place, making it easier to review and act upon.
+
+#### How to Use Comments Effectively
+
+1. **Adding Comments:** Go to the specific slide you want to add a comment. Go to 'Review' tab | 'New Comment' or right-click on the slide and select 'New Comment'. In 'Comments' pane on the right hand side , type your feedback or suggestion directly into the comment box.
+
+![](create-new-comment.png)
+
+
+
+2. **Replying to Comments:** If you're responding to feedback, use the 'Reply' feature within the comment to maintain a threaded conversation. This keeps the discussion organized and easy to follow.
+
+3. **Convert Comments to Tasks:** When reviewing comments, identify action items and assign them to team members by tagging them in the reply and ticking off the 'Assign to: xx' box.
```

Comment:
hey @synnoveisabel 

- [ ] 1. Can you change from 'xx' to '{{ ASSIGNEE }}' 

OR even just remove the 'xx' 

Up to you ;)